src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.utility.Parser;
+import duke.utility.Storage;
+import duke.utility.TaskList;
+import duke.utility.Ui;
+
+/**
+ * Class to represents the Duke.
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructs a new Duke object to initialized the program.
+     */
+    public Duke() {
+        ui = new Ui();
+        storage = new Storage("data/storage.txt");
+        try {
+            tasks = new TaskList(storage.loadTasksFromDisk());
+        } catch (DukeException e) {
+            ui.showError(e);
+            tasks = new TaskList();
+        }
+    }
+
+    public String getResponse(String input) {
```

Comment:
It would be good to have header comments for all your public classes and methods. However, I understand that this may not be included yet for your project.